File: Programming Tutorials/Trie.java
// Begin implementation of Tri

import java.util.*;

// Node for trie
class Node
{
	char cargo;
	Map<String, Node> next; // Dictionary

	Node(char item)
	{
		cargo = item;
		next = new HashMap<String, Node>();
	}

	Map getNext()
	{
		return next;
	}
}

class A_Trie
{
	private Node root;

	A_Trie()
	{
		root = new Node(' ');
	}

	void insert(char item)
	{
		Node newNode = new Node(item);

	}
}

public class Trie
{
	public static void main(String args[])
	{
		return;
	} 
}

File: Programming Tutorials/Trie.py
class Node(object):
	def __init__(self, item=None):
		self.cargo = item
		self.next = {}

class Trie(object):
	def __init__(self):
		self.root = Node()

	def insert(self, item):
		currentNode = self.root

		for i in item:
			if currentNode.next.hasKey(item[i]):
				currentNode = currentNode.next[item[i]]
			else:
				currentNode.next[item[i]] = Node(i)
				currentNode = currentNode.next[item[i]]

	def display(self):
		
def main():
	print "Hello"
	print "Why"

if __name__ == '__main__':
	main()